ansible/roles/slash/config/common/_slash_generator.py

```
@@ -0,0 +1,186 @@
+) -> None:
+    """Generate Markdown prompt files for Codex."""
+
+    clean_destination(destination)
+
+    for command in commands:
+        prompt_path = ensure_prompt_exists(prompt_root, command.prompt_file)
+        safe_key = command.key
+        if not all(ch.isalnum() or ch in {"_", "-", "."} for ch in safe_key):
+            raise SlashGeneratorError(
+                f"Invalid command key '{command.key}' (contains unsafe characters)."
+            )
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)